packages/components/src/Banner/Banner.stories.tsx

```diff
@@ -5,7 +5,6 @@ import Heading from "../Heading";
 import Banner from "./Banner";
 
 export default {
-  title: "Banner",
```

Comment:
Storybook can now generate titles for us, based on the directory structure.

We can still provide our own if we want, though.